### DISCUSSION TITLE
Unable to save no-mop zone

### DISCUSSION THREAD

**Discussion by jkms** [Hypfer/Valetudo#845] [Q&A / Support] [answered]
Model: S5 Max
Valetudo: 2021.03.0

First of all, it's not clear to me which button is which....
![image](https://user-images.githubusercontent.com/660916/115475783-444eb380-a1f5-11eb-9f24-0d1955d79573.png)
I'm assuming:
* the second button (square with an X) is the No-Go zone.  This creates a Red Box in the map
* the third button (circle with a square in it) is the No-Mop zone.  This creates a purple square in the map.

When I create a Red "No-Go" zone, I'm able to save the changes and see the zone on the map.  However, when I create a Purple "no-mop" zone, despite pressing the save button nothing shows on the map, and re-entering the zone-editing menu also is missing the Purple No-Mop zone.

What am I doing wrong?

Thanks (and amazing project, by the way.  So happy to liberate my vacuum and have such tight integration with Home Assistant).
**--John**

**Comment by Hypfer** [comment, marked as answer]
> First of all, it's not clear to me which button is which....

Yep. The icons are usless and all over the place. That will be adressed with the UI rewrite

> What am I doing wrong?

Editing no-mop zones is broken in 2021.03.0
That will be fixed in the next release

**Reply by jkms** [reply]
Easy-peasy.  Sit and wait.

Thanks.
